src/site/site.xml

```diff
@@ -28,8 +28,8 @@
 
     <menu name="Apache log4net" inherit="top" img="icon-home">
       <item name="About" href="/index.html"/>
```

Comment:
@FreeAndNil, how does this work at all? The root points to `logging.[staged.]apache.org`. Hence, `/index.html` will point to `l.a.o/index.html` instead of the expected `l.a.o/log4net/index.html`.

Reply:
As I understand, the root points to `logging.[staged.]apache.org/log4net`

Reply:
Checking the `target/site` output of the `mvn site` execution, figured `maven-site-plugin` replaces `/` with relative paths.